dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+
+Node
+`npm install @clayui/css`
+
+Yarn
+`yarn add @clayui/css`
+
+[Read more at Clay's website](https://clayui.com/docs/get-started/how-to-use-clay.html)
+
+## How to use Clay once imported?
+
+And finally we arrive at the section where we explain what to do with Clay once imported. In order to have Clay available in our CSS we'll need to import it in our `main.scss`, `style.scss` or `your-custom-name-css-file.scss`.
+
+It's a really simple code line:
+
+```scss
+@import 'atlas-variables';
+```
+
+Except, _it's not_ 😁.
+
+---
+
+There are 4 different ways to import Clay CSS.
+
+```scss
+@import 'base';
+@import 'base-variables';
+@import 'atlas';
+@import 'atlas-variables';
+```
+
+And, of course, each one of them has its unique characteristics.
+
+| Import          | Description                                                 |
+| --------------- | ----------------------------------------------------------- |
+| base            | All the functionalities of Clay with Bootstrap style        |
+| base-variables  | The main functions and variables of Clay in Bootstrap style |
+| atlas           | All the functionalities of Clay with Lexicon style          |
+| atlas-variables | The main functions and variables of Clay in Lexicon style   |
+
+[Learn everything about Clay CSS](https://github.com/liferay/liferay-frontend-guidelines/blob/281b28e93db2ade5a18230be8ac3b618f28cddb0/css/clay-css-3-intro.md)
+
+---
+
+There is a fifth _unofficial_ line that allow us to include a specific component's CSS we don't usually use it but if you are building something special you can use:
+
+```scss
+@import 'path-to-node_modules-clay-css/components/_buttons'; //not working example
+```
+
+## Which options do we have with Clay?
+
+There are 2 main ways to use Clay in our CSS.
+
+The first one is "read mode" where we can access to variables, mixins and functions of the framework
+
+Among the many variables available in Clay we'd like to highlight:
+
+- Color variables: `$primary`, `$secondary`, `$success`, `$success-l1`, ... 
+- Spacing variables: `$spacers`, `$grid-breakpoints`, `$container-max-widths`, ...
+- Typography variables `$font-family-serif`, `$h1-font-size`, `$display1-size`, ...
+
+Sources: [BS](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/_variables.scss), [Base](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/variables/_globals.scss), [Atlas](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/atlas/variables/_globals.scss)
+
+There are also useful mixins and functions:
+
+- [Breakpoints](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/mixins/_breakpoints.scss)
+- [Text-Truncate](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/mixins/_text-truncate.scss)
```

Comment:
We don't want to encourage the use of this one. It needs `display:block;` or `display:inline-block;max-width:100%;` to work.